docker/entrypoint.sh

```diff
@@ -5,11 +5,11 @@ set -e
 EXTERNAL_PEER_ADDRESS="${EXTERNAL_PEER_ADDRESS:-http://$(curl -s https://api.ipify.org):3013/}"
 
 # Allow changing peers address[0], defaults to testnet node1
-PEERS_ADDRESS_0="${PEERS_ADDRESS_0:-http://31.13.249.1:3013/}"
+PEERS_ADDRESS_0="${PEERS_ADDRESS_0:-peer: {host: 31.13.249.1, port: 3015, pubkey: pp\$2eDAWTgveKp1C4dWhy9Hg59NCrg8TPUCKSXeEgvnPdro4ra177}"
 
 # Use TestNet example config as configuration template
 sed -e "s|peer_address: http://127.0.0.1:3013/|peer_address: ${EXTERNAL_PEER_ADDRESS}|g" \
-    -e "s|http://31.13.249.1:3013/|${PEERS_ADDRESS_0}|g" \
+    -e "s|peer: {host: 31.13.249.1, port: 3015, pubkey: pp\$2eDAWTgveKp1C4dWhy9Hg59NCrg8TPUCKSXeEgvnPdro4ra177}|${PEERS_ADDRESS_0}|g" \
```

Comment:
Does this mean that all running Docker containers will by default have the same peer identity?

Reply:
Ignore this please.